lib/Autocomplete.js

```diff
@@ -298,8 +304,8 @@ let Autocomplete = React.createClass({
         state: this.state
       })
     }
-    return ( 
-      <div style={{display: 'inline-block'}}>
+    return (
+      <div {...this.props.wrapperProps} style={{...this.props.wrapperStyle}}>
```

Comment:
Perhaps `wrapperProps` should come last so the consumer can optionally specify `style` in `wrapperProps`?

e.g. `<Autocomplete wrapperProps={{ style: { ... }, ... }} />`